src/Servers/Kestrel/test/Interop.FunctionalTests/Http3/Http3RequestTests.cs

```diff
@@ -511,6 +511,57 @@ public async Task GET_ServerAbort_ClientReceivesAbort(HttpProtocols protocol)
             }
         }
 
+        [ConditionalFact]
+        [MsQuicSupported]
+        public async Task POST_Expect100Continue_Get100Continue()
+        {
+            // Arrange
+            var builder = CreateHostBuilder(async context =>
+            {
+                var body = context.Request.Body;
+
+                var data = await body.ReadAtLeastLengthAsync(TestData.Length).DefaultTimeout();
+
+                await context.Response.Body.WriteAsync(data);
+            });
+
+            using (var host = builder.Build())
+            using (var client = Http3Helpers.CreateClient(expect100ContinueTimeout: TimeSpan.FromMinutes(20)))
+            {
+                await host.StartAsync().DefaultTimeout();
+
+                var requestContent = new StreamingHttpContent();
```

Comment:
Consider replacing the StreamingHttpContent with a StringContent, streaming isn't relevant to this test and it adds extra code.